### PR TITLE
Add VerificationLevel value consts and a typedef for int

### PR DESCRIPTION
**Pull request by nstafie**
This time it'll work! :)

### CODE REVIEW

File: structs.go [hunk 1]
```diff
@@ -145,35 +145,46 @@ type Emoji struct {
 	RequireColons bool     `json:"require_colons"`
 }
 
+// Custom VerificationLevel typedef for int
+type VerificationLevel int
+
+// Constants for VerificationLevel levels from 0 to 3 inclusive
+const (
+	VerificationLevelNone   VerificationLevel = iota
```
Comment (iopred) [inline structs.go]: You can remove the = iota on every line after the first

Comment (nstafie) [inline structs.go]:
```go
const (
	VerificationLevelNone   VerificationLevel = iota
	VerificationLevelLow    VerificationLevel
	VerificationLevelMedium VerificationLevel
	VerificationLevelHigh   VerificationLevel
)
```

Like this?

**Comment by iopred** [comment]
Even better:

const (
    VerificationLevelNone   VerificationLevel = iota
    VerificationLevelLow
    VerificationLevelMedium
    VerificationLevelHigh
)

**Comment by nstafie** [comment]
Yeah that's what I ended up doing, the way I asked about won't even compile

**Comment by bwmarrin** [comment]
Yay :)

**Pull request merged**